PKGBUILD

```diff
@@ -1,37 +1,55 @@
 pkgdesc="C++ library for Franka Emika research robots "
 url="http://wiki.ros.org/libfranka"
 
-pkgname='ros-noetic-libfranka'
-pkgver='0.7.1'
-arch=('i686' 'x86_64' 'aarch64' 'armv7h' 'armv6h')
+pkgname="ros-noetic-libfranka"
+pkgver="0.8.0"
 pkgrel=1
+pkgdesc="C++ library for Franka Emika research robots "
+arch=('i686' 'x86_64' 'aarch64' 'armv7h' 'armv6h')
+url="http://wiki.ros.org/libfranka"
 license=('Apache')
 
-ros_makedepends=(ros-noetic-roscpp ros-noetic-catkin)
-makedepends=('cmake' 'ros-build-tools' ${ros_makedepends[@]})
+ros_makedepends=()
+makedepends=('cmake' 'git' ${ros_makedepends[@]})
 
 ros_depends=()
 depends=('eigen' 'poco' ${ros_depends[@]})
 
-_dir="libfranka-release-upstream-$pkgver"
-source=("$pkgname-$pkgver.tar.gz::https://github.com/frankaemika/libfranka-release/archive/upstream/$pkgver.tar.gz")
-sha256sums=("4e552542954f43b03d68e4c8f2b21f221ac563b40ad9d7e63dd9938a17db0ab7")
+source=(
+    "git+https://github.com/frankaemika/libfranka-common.git"
+    "git+https://github.com/frankaemika/libfranka.git#tag=$pkgver"
+    "fix_missing_includes.patch"
```

Comment:
For the patch instead of adding it locally if you could pull it in from upstream or make a PR upstream and pull it from there it would be best.